en/flight_modes_fw/return.md

```diff
@@ -0,0 +1,229 @@
+At the end there are sections explaining the _default_ (preconfigured) behaviour for each [vehicle type](#default_configuration).
+
+:::note
+
+- Mode is automatic - no user intervention is _required_ to control the vehicle.
+- Mode requires a global 3d position estimate (from GPS or inferred from a [local position](../ros/external_position_estimation.md#enabling-auto-modes-with-a-local-position)).
+  - Flying vehicles can't switch to this mode without global position.
+  - Flying vehicles will failsafe if they lose the position estimate.
+- Mode requires home position is set.
+- Mode prevents arming (vehicle must be armed when switching to this mode).
+- RC control switches can be used to change flight modes on any vehicle.
+- RC stick movement in a multicopter (or VTOL in multicopter mode) will [by default](#COM_RC_OVERRIDE) change the vehicle to [Position mode](../flight_modes_mc/position.md) unless handling a critical battery failsafe.
+
+<!-- https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/commander/ModeUtil/mode_requirements.cpp -->
+
+:::
+
+<a id="return_types"></a>
+
+## Return Types (RTL_TYPE)
+
+PX4 provides four alternative approaches ([RTL_TYPE](#RTL_TYPE)) for finding an unobstructed path to a safe destination and/or landing:
+
+- [Home/rally point return](#home_return) (`RTL_TYPE=0`): Ascend to safe altitude and return via a direct path to the closest rally point or home location.
+- [Mission landing/rally point return](#mission_landing_return) (`RTL_TYPE=1`): Ascend to a safe altitude, fly direct to the closest destination _other than home_: rally point or start of mission landing.
+  If no mission landing or rally points are defined, return home via direct path.
+- [Mission path return](#mission_path_return) (`RTL_TYPE=2`): Use mission path and fast-continue to mission landing (if defined).
+  If no mission landing defined, fast-reverse mission to home.
+  If no mission defined, return direct to home (rally points are ignored).
+- [Closest safe destination return](#safety_point_return) (`RTL_TYPE=3`): Ascend to a safe altitude and return via direct path to closest destination: home, start of mission landing pattern, or rally point.
+  If the destination is a mission landing pattern, follow the pattern to land.
+
+More detailed explanations for each of the types are provided in the following sections.
+
+<a id="home_return"></a>
+
+### Home/Rally Point Return Type (RTL_TYPE=0)
+
+In this return type the vehicle:
+
+- Ascends to a safe [return altitude](#return_altitude) (above any expected obstacles).
+- Flies via direct path to the home position or a rally point (whichever is closest)
+- [Land or waits](#arrival) at descent altitude (depending on landing parameters).
+
+:::note
+If no rally points are defined, this is the same as a _Return to Launch_ (RTL)/_Return to Home_ (RTH).
+:::
+
+<a id="mission_landing_return"></a>
+
+### Mission Landing/Rally Point Return Type (RTL_TYPE=1)
```

Comment:
Rally points landings are not possible yet in FW. They really only can be used to "rally" a/multiple vehicles at certain points on the map (they then just loiter and wait there).

Reply:
Proper rally point landing support is coming though